### ISSUE TITLE
AgentConnect Event is not raised

### ISSUE THREAD

**Issue by dleventidis** [AsterNET/AsterNET#287]
Hi again,

I have another issue with the `AgentConnectEvent` which is not raised as it was on my previous issue (#286)

Thanks in advance

**Comment by Deantwo** [comment]
I have never used the `AgentConnectEvent`, so I don't know what you are trying to do with it.

As I said in #286, you are using a very old version of Asterisk. Their documentation website doesn't even list your version anymore. so finding details on the AMI commands and events is difficult.
This is the best I can find is: https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/AMI_Events/AgentConnect/, and it might even have been changed in between Asterisk11 and Asterisk16.

I don't even know if the `AgentConnect` AMI event existed in Asterisk11, since I have never used it myself (and I used Asterisk11 and up back in the day).
